src/test/scala/org/kframework/minikore/TextToMiniTest.scala

```diff
@@ -237,45 +243,103 @@ class TextToMiniTest {
         |module A
         |  axiom \tr ue() []
         |endmodule []
-      """.stripMargin
+        |""".stripMargin
     try {
       parseFromString(s)
       assert(false)
     } catch {
-      case _: Throwable => ()
-        val msg =
+      case ParseError(msg) =>
+        assertEquals(
           """
             |ERROR: Line 4: Column 12: Expected 'u', but ' '
             |  axiom \tr ue() []
             |           ^
-          """.stripMargin
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
+    }
+  }
+
+  @Test def parseTestFail10(): Unit = {
+    val s =
+      """
+        |[]
+        |module A
+        |  axiom \t rue() []
+        |endmodule []
+        |""".stripMargin
+    try {
+      parseFromString(s)
+      assert(false)
+    } catch {
+      case ParseError(msg) =>
+        assertEquals(
+          """
+            |ERROR: Line 4: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't '
+            |  axiom \t rue() []
+            |          ^
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
+    }
+  }
+
+  @Test def parseTestFail11(): Unit = {
+    val s =
+      """
+        |[]
+        |module A
+        |  axiom \t
+        |""".stripMargin
+    try {
+      parseFromString(s)
+      assert(false)
+    } catch {
+      case ParseError(msg) =>
+        assertEquals(
+          """
+            |ERROR: Line 4: Column 11: Expected \true, \false, \and, \or, \not, \implies, \exists, \forall, \next, \rewrite, or \equal, but 't\n'
+            |  axiom \t
+            |          ^
+            |""".stripMargin.stripPrefix(System.lineSeparator()).stripSuffix(System.lineSeparator()),
+          msg)
```

Comment:
We should have 100% code coverage in minikore, but all coverage criteria.  So we'll need more tests, as we only have 92% coverage in some classes.